Extensions/AuthPropertiesTokenCacheHelper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Client;
+
+namespace TodoListService.Extensions
+{
+    public class AuthPropertiesTokenCacheHelper
+    {
+        private const string TokenCacheKey = ".UserTokenCache";
+
```

Comment:
Why have an empty line?

Reply:
@henrik-me What tool are you referring to?

Also, line has been removed.